### ISSUE TITLE
Improvement of writing UnitGenerateOptions

### ISSUE THREAD

**Issue by ryuix** [Cysharp/UnitGenerator#3]
Currently, UnitGenerateOptions definitions tend to be long as shown below.

```csharp
namespace Sample
{
    [UnitOf(typeof(int), UnitGenerateOptions.ArithmeticOperator | UnitGenerateOptions.ValueArithmeticOperator | UnitGenerateOptions.Comparable | UnitGenerateOptions.MinMaxMethod)]
    public readonly partial struct Hp { }
}
```

So, I want to write as follows.

```csharp
namespace Sample
{
    [UnitOf(typeof(int), (UnitGenerateOptions)60)]
    public readonly partial struct Hp { }
}
```

or

```csharp
namespace Sample
{
    internal static class Union
    {
        public const UnitGenerateOptions AVCM = UnitGenerateOptions.ArithmeticOperator | UnitGenerateOptions.ValueArithmeticOperator | UnitGenerateOptions.Comparable | UnitGenerateOptions.MinMaxMethod;
    }

    [UnitOf(typeof(int), Union.AVCM)]
    public readonly partial struct Hp { }
}
```

However, this is currently impossible.

## in Japanese

UnitGenerateOptionsの宣言が長くなりがちなので、下2つのような短縮した宣言を可能にしたいと考えています。

**Comment by neuecc** [comment]
ありがとうございます、それをやりたいと思っていた
＆いまいちやり方が浮かんでなかったので(PR見てなるほどです！）大変助かりました！

**Comment by neuecc** [comment]
I've released 1.2.0, it includes this fix.

**Comment by zacharylayne** [comment]
> Currently, UnitGenerateOptions definitions tend to be long as shown below.
> 
> ```cs
> namespace Sample
> {
>     [UnitOf(typeof(int), UnitGenerateOptions.ArithmeticOperator | UnitGenerateOptions.ValueArithmeticOperator | UnitGenerateOptions.Comparable | UnitGenerateOptions.MinMaxMethod)]
>     public readonly partial struct Hp { }
> }
> ```
> 
> So, I want to write as follows.
> 
> ```cs
> namespace Sample
> {
>     [UnitOf(typeof(int), (UnitGenerateOptions)60)]
>     public readonly partial struct Hp { }
> }
> ```
> 
> or
> 
> ```cs
> namespace Sample
> {
>     internal static class Union
>     {
>         public const UnitGenerateOptions AVCM = UnitGenerateOptions.ArithmeticOperator | UnitGenerateOptions.ValueArithmeticOperator | UnitGenerateOptions.Comparable | UnitGenerateOptions.MinMaxMethod;
>     }
> 
>     [UnitOf(typeof(int), Union.AVCM)]
>     public readonly partial struct Hp { }
> }
> ```
> 
> However, this is currently impossible.
> 
> ## in Japanese
> UnitGenerateOptionsの宣言が長くなりがちなので、下2つのような短縮した宣言を可能にしたいと考えています。

If you want shorter names in the attribute, use:
`using static UnitGenerator.UnitGenerateOptions;`
